provider/docker.go

```diff
@@ -34,6 +34,7 @@ type Docker struct {
 	Domain           string     `description:"Default domain used"`
 	TLS              *ClientTLS `description:"Enable Docker TLS support"`
 	ExposedByDefault bool       `description:"Expose containers by default"`
+	UseBindPortIP    bool       `description:"Use the ip address from the port binded instead of the inner network one"`
```

Comment:
The description could be a bit better here, the language feels a bit awkward...

Perhaps this would be better:

"Use the ip address from the bound port, rather than from the inner network"